packages/EasyErrorHandler/src/Bridge/Symfony/Resources/translations/EasyErrorHandlerBundle.en.php

```diff
@@ -8,6 +8,9 @@
         'bad_request' => 'Bad request.',
         'conflict' => 'Conflict.',
         'entity_not_valid' => 'Entity validation failed.',
+        'error_codes_provider' => [
```

Comment:
Pls remove it, it's not relevant anymore.

Reply:
Done.